content/docs/concepts/low-power.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Low Power"
+linkTitle: "Low Power"
+type: "docs"
+weight: 35
+description: >
+  How to consume less power on microcontrollers
+---
+
+Microcontrollers are usually rather non-demanding when it comes to power consumption.
+Couple of watts is the most you may expect a hobby project would use.
```

Comment:
I think milliwatts are more realistic, usually microcontrollers don't use more than 10mA or so, 100mA in peaks maybe.

Also, perhaps amperes (milliamperes) are more useful here? When I see power consumption in datasheets and such, it is usually measured in milliamperes not in watts (their voltage is constant, or the current consumption even goes _up_ on a higher voltage - in that case current is more precise).

Reply:
I was more thinking about a hobby project as a whole, with sensors, leds, relays, servos, etc.
But yeah, we can be less dramatic here. "couple hundreds mA" shall be fine, right?

Reply:
As for watts vs. amps... if you say amps -- the next question is: how many volts?
Also, people are used to watts, in light bulbs, in heating elements, on laptop chargers. And at the end of a month, when it's time to see kWh (oh my god).
But yeah, batteries are in mAh and we are measuring amps. So probably I mention watts in parentheses first time and then switch to amps entirely.

Reply:
Removed watts from everywhere but the second sentence. It starts now "Couple hudreds mW"